tests/unit/specs/components/common/__snapshots__/TmSessionExplore.spec.js.snap

```diff
@@ -132,7 +132,7 @@ exports[`TmSessionExplore shows a form to sign in with an address 1`] = `
               class="tm-li-session-title"
             >
               <span>
-                cosmos1vxkye…a7m64khj8xyz
+                cosmos15ky9d…rm5ctpesxxn9
```

Comment:
?

Reply:
why did we change the format to a much longer one?

Reply:
ups, this is still not a valid bech32 address